juego.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, level, fuente):
             else:
                 wrong = str(int(self.resultado) + random.randint(1,10))
             image_wrong = fuente.render(wrong,True,(random.randint(0,255),random.randint(0,255),random.randint(0,255)))
-            self.preguntas.add(number(random.randint(300,1000),random.randint(-0,-0),image_wrong,False))
+            self.preguntas.add(number(random.randint(int(sx(300)),int(sx(1000))),random.randint(-0,-0),image_wrong,False))
```

Comment:
@chimosky this is in reply to your [comment](https://github.com/sugarlabs/jamath-activity/pull/14#issuecomment-718007950)
> The image on the activity canvas also extends the screen, with one part of the image being visible.

This shouldn't create a problem as it's just used as a background.

> Noticed some too, a number was too far to the right of the screen, it was visible but a tiny part of it wasn't.

Ohh so 30f77ce doesn't fix it then. Another option is to change the hardcoded coordinates. I don't see a pattern/ the exact calculations for using the coordinates which were used now. Reducing the x-coordinate from 1000 to a smaller number maybe 900?

Reply:
> This shouldn't create a problem as it's just used as a background.

Yeah it doesn't create any problem, I was thinking if something can be done about it.

> Reducing the x-coordinate from 1000 to a smaller number maybe 900?

Might work, you should try it.

Reply:
> Might work, you should try it.

In 1f42216 I changed it to 900, works perfectly for me.

> Yeah it doesn't create any problem, I was thinking if something can be done about it.

While fixing this I realized load_image was a duplicate function, the `cargar_imagen` function loads and scales the image. Fixed it here: 4e22003